_rules/SC3-1-2-lang-valid.md

```diff
@@ -88,6 +92,18 @@ The `lang` and `xml:lang` attribute specified has a non-empty value & a valid pr
 </html>
 ```
 
+#### Passed example 5
+
+The `lang` and `xml:lang` attributes values are either the empty string ("") or has a valid primary language subtag.
```

Comment:
The `lang` and `xml:lang` attributes values are either the empty string ("") or have a valid primary language subtag.